python/res/fm/util/template_render.py

```diff
@@ -0,0 +1,79 @@
+gen_kw_export_file = "parameters";
+
+import jinja2
+import json
+import os
+import yaml
+
+#default export file name found in enkf_defaults.hpp
+DEFAULT_GEN_KW_EXPORT_NAME = "parameters.json"
```

Comment:
Did you consider prototyping the name in `enkf_defaults.hpp`, so that you do not have to specify it twice?